<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonSettingsRandom</class>
 <widget class="QWidget" name="CommonSettingsRandom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView_common_random">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>791</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_common_random">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst DB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Typ vibr. profilu:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shocks</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vyhledat podle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> číslo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zakázky</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_contract">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_inqr">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_id">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contract_number">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> rok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contract_year">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> rok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_year">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit_common">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrdit změny</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Smazat záznam(y)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drag_profile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Přetáhnout profil</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_common_random_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_new_record">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nový záznam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete_new_record">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Smazat záznam(y)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit_new_record">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrdit nový profil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zavřít</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_revert_common">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zrušit změny</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK_filter_common">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>poptávky</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>index</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_index">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> číslo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>CommonSettingsRandom</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
